lib/amigo.rb

```diff
@@ -57,6 +57,10 @@
 # The 'on' pattern can be 'myapp.customer.*' to match all customer events for example,
 # or '*' to match all events. The rules of matching follow File.fnmatch.
 #
+# The 'on' pattern accepts regular expressions '/myapp\.customer\.created&/' to only match specific
```

Comment:
`The 'on' pattern also accepts regular expressions, like /^myapp\.customer\.[a-z]+$/, to control the matching rules more closely than `File.fnmatch` can provide.`